microcosm_fastapi/conventions/enums.py

```diff
@@ -0,0 +1,32 @@
+"""
+Enum conventions
+
+"""
+from enum import Enum
+from typing import Any, Dict
+
+
+class BaseEnum(Enum):
```

Comment:
I think it should live in marquez, right?
Do we need to update all our enums to inherit from this?

Reply:
Nice - have moved to marquez.  Yep will all "nested" enums need to inherit from this to prevent issues with json deserialisation